### PR TITLE
Allow for enabling Maker APIs within existing App Lab projects via URL parameter.

### PR DESCRIPTION
**Pull request by bcjordan**
This PR:

* Adds a `makerAPIsEnabled` field to project sources, populated by `level.makerlabEnabled` on saving.
* Adds `?enableMaker` and `?disableMaker` URL params which can be used to enable or disable maker APIs on a given project (eventually this will be triggered by a UI element)
* Now always includes maker dependency code and maker block definitions in App Lab, conditionally hiding the blocks and autocomplete (as is done with firebase's `onRecordEvent`).

TODO:

- [x] Confirm app bundle size hasn't changed significantly
  - [x] perform full build on staging, measure applab and common bundle sizes
  - [x] perform full build on applab-maker-enable and measure applab and common bundle sizes
- [x] Make sample p/makerlab project on staging
  - [ ] merge to staging
  - [ ] observe behavior for existing maker projects
- [ ] Find opportunities for tests to cover behavior change.

### CODE REVIEW

**Comment by bcjordan** [comment]
@islemaster could use an early review of this approach while I work on remaining wrap-up steps, PTAL!

File: apps/src/makerlab/BoardController.js [hunk 1]
```diff
@@ -3,20 +3,12 @@
...
-  /**
-   * These packages should currently only be downloaded and available on
-   * /p/makerlab levels, not vanilla App Lab ones, so we ignore any require
-   * failures here.
-   */
-}
+// Polyfill node's process.hrtime for the browser, gets used by johnny-five.
+process.hrtime = require('browser-process-hrtime');
+
+import five from 'johnny-five';
```
Comment (islemaster) [inline apps/src/makerlab/BoardController.js:9]: I think the import statement will get hoisted above the `hrtime` require here.  Is that ordering going to be a problem?  If so, you may need to leave `johnny-five` as a `require`.

**Comment by islemaster** [comment]
Looking really good so far!

File: apps/src/makerlab/BoardController.js (hunk 1 above)
Comment (bcjordan) [reply apps/src/makerlab/BoardController.js:9]: Confirmed this still works. (Will also try without it to make sure it's still required)

Comment (bcjordan) [reply apps/src/makerlab/BoardController.js:9]: (Indeed still required)

**Pull request merged**